Xbim.Geometry.Engine.Interop/XbimGeometryEngine.cs

```diff
@@ -35,6 +35,7 @@ public XbimGeometryEngine(ILogger<XbimGeometryEngine> logger)
             _logger = logger ?? XbimLogging.CreateLogger<XbimGeometryEngine>();
 
             //  Warn if runtime for Engine is not present, this is not necessary any more as we are net47
+            //XbimPrerequisitesValidator.Validate();

```

Comment:
There's commented out code, with obsolete comments above. The comment says it's not necessary anymore, so this can probably be removed completely.